### DISCUSSION TITLE
Multilanguage fields for Units

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#176] [Q&A] [answered]
Hi,
What is the reason that _Attributes/New_ Attribute Type: **Unit** has not **Multi-Language" Checkbox - while others have one?

![image](https://user-images.githubusercontent.com/1337223/105089101-fe227080-5a9c-11eb-930f-3eba214aa025.png)

I can't find a way to define that in the backend. And the only reference where multilang fields are defined are here: https://github.com/atrocore/atropim/blob/8fd4f51c62fa346837ca933f9459cf64f0b123b1/app/Module.php#L47 - but changing it didn't do much, so I guess I'm looking at the wrong place.

Thanks for any hint.

**Comment by atrolex** [comment, marked as answer]
It is not that simple. Front-end should be adjusted as well.
We will implement this feature in 2021 Q1, it is on our roadmap.

**Reply by marcus-at-localhost** [reply]
Ah I see, I was able to add the `unit` to the `$multiLangTypes` array and make the checkbox show up (didn't clear caches in my first test), but it's missing the correct UnitType for each entry.

![image](https://user-images.githubusercontent.com/1337223/105162560-146b1380-5b13-11eb-92d9-5d908c2910bd.png)

Thanks for clarifying.
